qtranslate_frontend.php

```diff
@@ -330,7 +330,7 @@ function qtranxf_excludeUntranslatedPosts($where,&$query) {//WP_Query
 	//qtranxf_dbg_echo('qtranxf_excludeUntranslatedPosts: post_type is empty: $query: ',$query, true);
 	//qtranxf_dbg_echo('qtranxf_excludeUntranslatedPosts: $where: ',$where);
 	//qtranxf_dbg_echo('qtranxf_excludeUntranslatedPosts: is_singular(): ',is_singular());
-	$single_post_query=is_singular();
+	$single_post_query=$query->is_singular();
```

Comment:
Sorry this commit is not related.
However, it solved a specific issue I had recently with mqTranslate.
Take for an example a page with a custom template inside which you set a second query using WP_Query. The is_singular() function is related to the main query recovering the page and not the WP_Query one. By doing this, is_singular() is always called for the good query.